1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,67 @@
-// Write your code here
+let images = [
```

Comment:
setting your image sources in an array & updating the img src in the dom is a great approach. However, your images aren't displaying properly as the images folder hasn't been uploaded